src/tests/e2e/pages/accounts.page.js

```diff
@@ -0,0 +1,48 @@
+import log from 'loglevel'
+log.setDefaultLevel('info')
+
+class AccountsManagementPage {
+  async getAccountsList () {
+    const accountsElementList = await page.$$('[data-test-id^="account_list_item"]')
```

Comment:
is "^" intentional?

Reply:
yes, select all data-test-id start with account_list_item